### DISCUSSION TITLE
zotprime-zotprime-minio-1  port number

### DISCUSSION THREAD

**Discussion by GuShaocheng** [uniuuu/zotprime#22] [Q&A]
Hi,
I installed server via docker compose up -d in linux:

error response from daemon: driver failed programming external connectivity on endpoint zotprime-zotprime-minio-1 (7399d39e02ade5f93d317e263686d1375ba6479dec8cd2b8f2acfd8bbe651305): Bind for 0.0.0.0:9000 failed: port is already allocated

Could you please show some idea to specify related port number? Thank you.
Besides, I wonder how to change the default password for zotero APT (admin,admin).

$ cat .env

## Ip/host of the Dataserver. 
DSHOST=http://10.8.0.1:8880/
STHOST=http://10.8.0.1:8881/

## SECRETS
#DB
MYSQLROOTPASSWORD=zotero
MYSQLDATABASE=zotprimeprod
MYSQLUSER=zotprimeprod
MYSQLPASSWORD=zotprimeprod

#MINIO
MINIOROOTUSER=zotero
MINIOROOTPASSWORD=zoterodocker

#Version 
VER=v2.7.0

# Ip/host of the Minio. 
S3HOST=10.5.5.1:9010

**Comment by uniuuu** [comment]
Hi @GuShaocheng 
`Bind for 0.0.0.0:9000 failed: port is already allocated`
indicates that at 9000 port there is already some app listening.
Could you please provide output of next command below?
`netstat -ntulp` or `ss -ntulp`

**Comment by uniuuu** [comment]
The issue/question is a support type.
Moving to discussions.

**Comment by uniuuu** [comment]
> Hi @GuShaocheng `Bind for 0.0.0.0:9000 failed: port is already allocated` indicates that at 9000 port there is already some app listening. Could you please provide output of next command below? `netstat -ntulp` or `ss -ntulp`

Hi @GuShaocheng 
Did it work out for you?

**Comment by uniuuu** [comment]
Hi @GuShaocheng 
I hope you resolved the issue. Please reopen discussion if more help needed.
